app/src/main/java/com/eis/phoneringer/structure/AppManager.java

```diff
@@ -0,0 +1,98 @@
+package com.eis.phoneringer.structure;
+
+import android.content.Context;
+import android.media.Ringtone;
+import android.os.Build;
+import android.os.Handler;
+import android.os.Looper;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+
+import com.eis.smslibrary.SMSHandler;
+import com.eis.smslibrary.exceptions.InvalidSMSMessageException;
+import com.eis.smslibrary.exceptions.InvalidTelephoneNumberException;
```

Comment:
You're importing classes from package com.eis.smslibrary, but you didn't include it as a dependency in module `app`'s `build.gradle`.